store/helper/helper.go

```diff
@@ -587,7 +603,13 @@ func (h *Helper) GetRegionsTableInfo(regionsInfo *RegionsInfo, schemas []*model.
 	tables := []tableInfoWithKeyRange{}
 	for _, db := range schemas {
 		for _, table := range db.Tables {
-			tables = append(tables, newTableWithKeyRange(db, table))
+			if table.Partition != nil {
```

Comment:
Why changing the codes here? Looks that it is not related to the 'privileges'.

Reply:
I will remove the code here. Seems that it is caused by I checkout with the other branch.

Reply:
I have removed it. PTAL